### ISSUE TITLE
More thinking on logo

### ISSUE THREAD

**Issue by bollwyvl** [robots-from-jupyter/robotlab#23]
Not super high value for the workshop, but how does this hit you?
![lab_new svg](https://user-images.githubusercontent.com/45380/51180875-9fb3cd80-1897-11e9-9649-6aa0ae450ee5.png)

**Comment by datakurre** [comment]
Maybe if we got permission to use Robot Framework logo instead of R in that bottle.

Just "R" has more popular meaning in Jupyter context.

**Comment by bollwyvl** [comment]
Yeah, it's all very hairy trying to mix logos .. but didn't get anything
else if merit done while traveling!

The RF logo has some siding guidelines that might make it hard to combine
with anything anyway. Anyhow, like I said, not worth worrying about!

**Comment by datakurre** [comment]
How about something like an item from periodic table? That could be pretty close with the current one.

**Comment by bollwyvl** [comment]
Good idea!

Rubinium, radon... But due to how whitespace works, just the 🤖 face would
be good.
